tests/fixtures/bad_conf.toml

```diff
@@ -1 +1,5 @@
+# SPDX-FileCopyrightText: 2024 Red Hat, Inc
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
The files in this directory have been added to `REUSE.toml`, no need for these changes anymore.